tools/js-sdk-release-tools/src/hlc/generateMgmt.ts

```diff
@@ -25,11 +25,12 @@ export async function generateMgmt(options: {
     additionalArgs?: string;
     outputJson?: any;
     swaggerRepoUrl?: string;
+    downloadUrlPrefix?: string;
     runningEnvironment?: RunningEnvironment;
 }) {
     logger.logGreen(`>>>>>>>>>>>>>>>>>>> Start: "${options.readmeMd}" >>>>>>>>>>>>>>>>>>>>>>>>>`);
 
-    let cmd = `autorest --version=3.7.3 --typescript --modelerfour.lenient-model-deduplication --azure-arm --head-as-boolean=true --license-header=MICROSOFT_MIT_NO_VERSION --generate-test --typescript-sdks-folder=${options.sdkRepo} ${path.join(options.swaggerRepo, options.readmeMd)}`;
+    let cmd = `autorest --version=3.8.4 --typescript --debug --modelerfour.lenient-model-deduplication --azure-arm --head-as-boolean=true --license-header=MICROSOFT_MIT_NO_VERSION --generate-test --typescript-sdks-folder=${options.sdkRepo} ${path.join(options.swaggerRepo, options.readmeMd)}`;
```

Comment:
So we'll enable `debug` by default? I know it will introduce some debug files and logs when generating samples if enable this option.

Reply:
yes. it's fraustructed when we encounter  unexpected error and there is no debug information. So I prefer to add it.

Reply:
don't like the idea of put --debug here. It will generate a lot of useless information, which makes it even harder to find the error message.

Reply:
testmodeler will also generate the code model yaml file if you put --debug here,

Reply:
okay, removed.

Reply:
> don't like the idea of put --debug here. It will generate a lot of useless information, which makes it even harder to find the error message.

I refactor the user experience in finding error message. Now, you can find them in spec PR directly in most times. (If the error is not output to stderr, I cannot show them, but I think this scenario is rare.)